todo-app/src/Todos.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+export default class TodoItems extends Component {
+
+    render() {
+        const todoItems = this.props.todos;
+        
+        return (
+            <ul>
+                {todoItems.map((item, i) => {
+                    return <li key={i}>
+                        {item.task}, {item.deadline}
+                    </li>
+                })}
+            </ul>    
+        );
+    }
+}
+
+
+TodoItems.propTypes = {
```

Comment:
Nice use of prop validation! If you were using eslint, you might see a warning like:

> todos is not marked as required, and no default value is provided

Or, in other words: "what would happen if this.props.todos was undefined"? You can either set default props or mark this prop as `.isRequired`.

You can check out the [eslint rule here](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md). This is a little bit more advanced, but something to be aware of!